go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/submariner-io/submariner-operator
 
-go 1.13
+go 1.16
```

Comment:
Hehe, I guess it's just for the embedded file.

It's fine as I see we have another PR planning to do that on a deeper level and tests are passing: https://github.com/submariner-io/submariner-operator/pull/1400/files